src/Blt/Plugin/Commands/SwsCommands.php

```diff
@@ -182,7 +182,8 @@ public function updateEnvironment($environment_name, $options = [
     'rebuild-node-access' => FALSE,
     'database-only' => FALSE,
     'configs-only' => FALSE,
-  ]) {
+  ]
+  ) {
```

Comment:
Is this a linter change and on purpose? Curious about the formatting here.

Reply:
I see a few formatting changes in this PR.

Reply:
Yeah, that's my PHPStorm doing it's thing. I normally format just a small area, but I didn't do it correctly this time. I was debating reverting those parts, but figured I'd see what you're thoughts are

Reply:
I find it a little confusing to have two "ending" characters at the same indentation on two separate lines. This would be less confusing to me I think? I'm not sure though because of how it opens. 
```suggestion
    ]
  ) {
```

Reply:
Is this possible in a function:
```
public function updateEnvironment($environment_name, $options = 
    [
      'rebuild-node-access' => FALSE,
      'database-only' => FALSE,
      'configs-only' => FALSE,
    ]
  ) {
  ```
  Of course this feels like it's too big/using too many lines.

Reply:
I don't _really_ have an opinion though 😆

Reply:
I reverted all the junk.